### PR TITLE
Add sarpy

### PR DESCRIPTION
**Pull request by jordan-heemskerk**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by jordan-heemskerk** [comment]
@conda-forge/help-python can I get a review on this, thanks!

File: recipes/sarpy/LICENSE.txt [hunk 1]
```diff
@@ -0,0 +1,21 @@
+MIT License
```
Comment (dopplershift) [inline recipes/sarpy/LICENSE.txt:1]: The PyPI tarball used here already includes `LICENSE`, so you shouldn't add this manually. Please remove.

File: recipes/sarpy/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://github.com/ngageoint/sarpy
+  summary: Python tools for reading, writing, and simple processing of complex SAR data and other associated data.
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE.txt
```
Comment (dopplershift) [inline recipes/sarpy/meta.yaml:41]: Update to the one in the tarball:
```suggestion
  license_file: LICENSE
```

**Review by dopplershift — CHANGES_REQUESTED** [review]
Just some minor tweaks to the license handling and this is good to go.

**Comment by jordan-heemskerk** [comment]
@dopplershift updated the license stuff, my bad.

Let me know if you need anything else.

**Review by dopplershift — APPROVED** [review]
Looks good pending CI.

**Pull request merged**